docs/list.md

```diff
@@ -148,6 +148,36 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
     - 4 merged PRs fixing a `Hacktoberfest` Issue = T-shirt and $100 credit.
     - More information [here](https://twitter.com/chimoney_io/status/1576286913723596803?s=48&t=JCGsKCKdKEB_sVfGml2m6w){:target="\_blank"} and on their [Community Repo](https://github.com/Chimoney/chimoney-community-projects){:target="\_blank"}.
 
+#### **Clueless**
+
+- **Swag**: T-shirt, certificate for contributing
+- **Requirements**: Contribute to the following repositories:
+    - [Clueless Community Website](https://github.com/Clueless-Community/clueless-official-website){:target="_blank"}
+    - [First-Contribution](https://github.com/Clueless-Community/first-contribution){:target="_blank"}
+    - [College API](https://github.com/Clueless-Community/collegeAPI){:target="_blank"}
+    - [Prega.io](https://github.com/Clueless-Community/Prega){:target="_blank"}
+    - [Datasets](https://github.com/Clueless-Community/Datasets){:target="_blank"}
+    - Earn points for your contributions:
```

Comment:
Can you please point to where you see info regarding these points? I'm having a hard time verifying this part of the entry. What are the points for? The shirt to the top contributors?

Reply:
The point system and top contributor part is provided on their org github page 
https://github.com/Clueless-Community#how-points-are-distributed

I know, the details for the event are scattered all over the place 😆

Reply:
> The point system and top contributor part is provided on their org github page 
> 
> https://github.com/Clueless-Community#how-points-are-distributed
> 
> 
> 
> I know, the details for the event are scattered all over the place 😆 

Yep. I am glad I went through the repository! That's why we have this awesome repository to consolidate all the information in one place.

@crweiner and @vimode we are doing cool stuff :)

Reply:
Ahh, thank you. So it's just a leaderboard, like SendGrid used to have? That Readme says the ["Top contributors will be winning amazing t-shirt and swags](https://github.com/Clueless-Community#win-amazing-swags).", so it sounds like there will be a points requirement of some sort for the shirt. I guess they haven't figured out a points threshold yet? If you happen to see a points requirement for the shirt anywhere, @kev-odin  or @vimode, please let me know!

Reply:
- One has to signup to their website and create a profile.
- When your PR is merged, login to the clueless website > get your profile link by going to your profile page.
- Comment in the merged PR your profile link and tag the maintainer who merged.
- This will update your rankings on the clueless website.

Top ranked contributors at the end of the month will win the prizes.

Heres an example merged PR https://github.com/Clueless-Community/clueless-official-website/pull/217

Reply:
Right. I think it is a contest for contributions. Not sure if there a point requirement for the shirt as it will be determined at the end. I will periodically check in with them. 

I need more coffee or need to stop fixing merge conflicts at 1 am lol